docs/game-servers/install-a-half-life-2-deathmatch-dedicated-server-on-debian-or-ubuntu.md

```diff
@@ -121,7 +121,7 @@ This script automatically starts your server in a Screen session.
 
         ./run.sh
 
-## Configure the server
+## Configure the Steam Server
```

Comment:
Should this H2 now be "Configure the Half-Life 2 Server" if server is NOT accurately described as a Steam Server?

Reply:
Yes, I think so.